README.md

```diff
@@ -1,13 +1,23 @@
-| Devtools | [![Quality Gate Status](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.devtools.karaf&metric=alert_status)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.devtools.karaf) | [![Code Smells](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.devtools.karaf&metric=code_smells)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.devtools.karaf) | [![Coverage](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.devtools.karaf&metric=coverage)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.devtools.karaf) |
-| OBR | [![Quality Gate Status](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.extensions.obr&metric=alert_status)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.extensions.obr) | [![Code Smells](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.extensions.obr&metric=code_smells)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.extensions.obr) |  |
-| Eclipse Parent | [![Quality Gate Status](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Agalasa-eclipse-parent&metric=alert_status)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Agalasa-eclipse-parent) | [![Code Smells](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Agalasa-eclipse-parent&metric=code_smells)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Agalasa-eclipse-parent) |  |
-| Eclipse Base Plugin | [![Quality Gate Status](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.eclipse&metric=alert_status)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.eclipse) | [![Code Smells](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.eclipse&metric=code_smells)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.eclipse) | [![Coverage](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.eclipse&metric=coverage)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.eclipse) |
-| Eclipse Base Feature | [![Quality Gate Status](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.eclipse.feature&metric=alert_status)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.eclipse.feature) | [![Code Smells](https://sonarqube.galasa.dev/api/project_badges/measure?project=dev.galasa%3Adev.galasa.eclipse.feature&metric=code_smells)](https://sonarqube.galasa.dev/dashboard?id=dev.galasa%3Adev.galasa.eclipse.feature) |  |
+This respository contains the Eclipse plug-in that enables you to run a Galasa test against SimBank or against your own application. 
```

Comment:
Need to remove the reference to simbank,  that's not the purpose of this repo or the eclipse plugin.

Reply:
Have removed reference to Simbank